test/cqlpy/test_secondary_index.py

```
@@ -2393,3 +2395,113 @@ def test_regular_row_update_with_static_set_index(cql, test_keyspace):
+
+            # The index has now diverged from the base: it claims (1,1,1,1)
+            # has v=3, the base says v=5. The queries must still be correct -
+            # a returned row must match the WHERE clause.
+            assert sorted(tuple(r) for r in cql.execute(q_in)) == expected
+            assert sorted(tuple(r) for r in cql.execute(q_range)) == expected
+        finally:
+            for err in (pause_scan, pause_apply, fail_update):
+                try:
+                    inj_disable(err)
+                except Exception:
+                    pass
```

Ruff (0.16.1)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)